### PR TITLE
Add project description to Readme file

### PR DESCRIPTION
**Pull request by DEz-1996**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,92 @@
...
+2. Order Management: Once you've populated your shopping cart with the books you crave, our application furnishes a streamlined order management system. You can scrutinize your orders and include them.
+3. Book Management with Pagination and Sorting: We recognize the significance of effective book exploration. Our application presents a sophisticated search functionality that empowers users to discover all books by categories, author, title, or by id. With pagination support, you can navigate through an extensive catalog of books without inundating your search results.
+4. Security and JWT Tokens for User Auth: Security is paramount in our application. We've integrated Spring Security to safeguard your data and transactions. JWT (JSON Web Tokens) are employed for secure authentication and authorization, guaranteeing that only authorized users can access sensitive functionalities.## Technologies used
```
Comment (yevheniisynytsia) [inline README.md]:
```suggestion
4. Security and JWT Tokens for User Auth: Security is paramount in our application. We've integrated Spring Security to safeguard your data and transactions. JWT (JSON Web Tokens) are employed for secure authentication and authorization, guaranteeing that only authorized users can access sensitive functionalities.
## Technologies used
```

File: README.md [hunk 2]
```diff
@@ -1 +1,92 @@
-# online-book-store
+# Online Book Store
```
Comment (yevheniisynytsia) [inline README.md:1]: please add an instruction how to run the application locally

**Review by yevheniisynytsia — CHANGES_REQUESTED** [review]
Great job! Let's add `how to tun the app` part!

**Review by d1sam — APPROVED** [review]
well done!

**Pull request merged**